man/pdfpc.in

```diff
@@ -70,6 +70,9 @@ Allow for testing pdfpc without proper installation. Icons, configuration files,
 etc. will be loaded from the source path locations. Mostly intended for
 developers.
 .TP
+.BI "\-p, \-\-rest\-port"
+Port the REST server binds to. (Default: 8088)
```

Comment:
Is it on purpose to use "radan http" port? https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=8088

Reply:
No, not really. I've never heard about it, frankly. Could be any other, as long as it is an unprivileged port and not typically used on an average PC...